### PR TITLE
Removed vehicle categories Sedan and Comapacts, Updated Vehicle Price…

### PR DESCRIPTION
**Pull request by Xrez31**
Removed vehicle categories Sedan and Comapacts, Updated Vehicle Prices, Changed Stretch category to super

### CODE REVIEW

**Comment by alvinveroy** [comment]
This needs to be put in a new SQL file. Modifying existing SQL file will result in an error in the database query.

**Review by alvinveroy — COMMENTED** [review]
Requires new sql file. This is not like a script where you can modify. These are all database queries which will result in "duplicates".

**Comment by Xrez31** [comment]
Added esx_updatedvehshop#1.sql

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Syntax error.

**Comment by Xrez31** [comment]
Removed excess 'or' on line 17

**Comment by Xrez31** [comment]
Fixed DELETE syntax for esx_updatedvehicleshop#1.sql

File: esx_updatedvehshop#1.sql [hunk 1]
```diff
@@ -1,21 +1,8 @@
...
-model='stromberg' or
-model='deluxo' or
-model='ardent' or
-model='oppressor' or
-model='blazer5' or
-model='ruiner2' or
-model='voltic' or
-model='monster'
-;
+DELETE FROM 'vehicles' WHERE model IN ('rumpo', 'stromberg', 'deluxo', 'ardent', 'oppressor', 'blazer5', 'ruiner2', 'voltic', 'monster');
```
Comment (alvinveroy) [inline esx_updatedvehshop#1.sql]: Syntax error use apostrophe instead of quotes in table name "vehicles" and "vehicle_categories"

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Syntax error.

File: esx_updatedvehshop#1.sql [hunk 2]
```diff
@@ -1,10 +1,10 @@
 USE `es_extended`;
 
-DELETE FROM 'vehicle_categories' WHERE name IN ('sedans', 'compacts');
+DELETE FROM `vehicle_categories` WHERE name IN ('sedans', 'compacts');
 
-DELETE FROM 'vehicles' WHERE model IN ('rumpo', 'stromberg', 'deluxo', 'ardent', 'oppressor', 'blazer5', 'ruiner2', 'voltic', 'monster');
+DELETE FROM `vehicles` WHERE model IN ('rumpo', 'stromberg', 'deluxo', 'ardent', 'oppressor', 'blazer5', 'ruiner2', 'voltic', 'monster');
 
-UPDATE 'vehicles'
+UPDATE `vehicles`
```
Comment (alvinveroy) [inline esx_updatedvehshop#1.sql:7]: This is a invalid SQL statement.

See: https://stackoverflow.com/questions/3667439/setting-multiple-column-using-one-update

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Invalid statement on UPDATE

**Comment by Xrez31** [comment]
Fixed UPDATE syntax

File: esx_updatedvehshop#1.sql [hunk 3]
```diff
@@ -26,7 +24,26 @@ SET price=
...
+	'sheava', 
+	'fmj', 
+	'infernus', 
+	'osiris', 
+	'pfister811', 
+	'le7b', 
+	'neon'
+);
 
 UPDATE 'vehicles' SET category='super' where model='stretch';
```
Comment (alvinveroy) [inline esx_updatedvehshop#1.sql]: Syntax error. Use apostrophe on table names.

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Syntax error, use apostrophe on table name.

**Pull request merged**